src/util/mongo.controller.js

```diff
@@ -37,10 +37,11 @@ const findResourceById = async (id, resourceType) => {
  */
 const findOneResourceWithQuery = async (query, resourceType) => {
   const collection = db.collection(resourceType);
-  return collection.findOne(query);
+  return collection.findOne(query, { projection: { _id: 0 } });
 };
 
-const findResourcesWithQuery = async (query, resourceType, options = {}) => {
+const findResourcesWithQuery = async (query, resourceType, options = { projection: { _id: 0 } }) => {
+  options.projection['_id'] = 0;
```

Comment:
I am not sure if we need to pass in the projection and then set it again? Honestly seems like we could probably even get rid of the `options = {}` input (I know that was already there but looks like it isn't even needed lol) and then you can just do `const results = collection.find(query, { projection: { _id: 0 } });` unless I am missing something (very possible).